bundles/aero.minova.rcp.rcp/src/aero/minova/rcp/rcp/handlers/PrintIndexHandler.java

```diff
@@ -177,40 +191,166 @@ public void createFile(String path) {
 	 * @param fileName
 	 * @param title
 	 */
-	private void saveIntoXML(SortedList<Row> rows, List<ColumnInfo> colConfig, List<Integer> columnReorderList, StringBuffer xml, boolean tabSeparated,
-			String fileName, String title) {
-		if (xml != null && rows != null && rows.iterator().hasNext()) {
-			int colIndex = 0;
-			addHeader(xml, fileName);
-			xml.append("<Title>" + title + "</Title>\n");
-			xml.append("<IndexView>\n" + "<Group>\n" + "<Text><![CDATA[Gesamt]]></Text>");
-			xml.append("<Rows>\n");
-			for (final Row r : rows) {
-				colIndex = 0;
-				xml.append("<Row>\n");
-				for (final Integer d : columnReorderList) {
-					Column c = colConfig.get(colIndex).column;
-					xml.append("<" + translationService.translate(c.getLabel(), null).replaceAll("[^a-zA-Z0-9]", "") + ">");
-					if (r.getValue(d) != null) {
-						if (r.getValue(d).getType() == DataType.DOUBLE || r.getValue(d).getType() == DataType.INTEGER) {
-							xml.append(r.getValue(d).getValueString(Locale.getDefault()));
-						} else if (r.getValue(d).getType() == DataType.BOOLEAN && r.getValue(d).getBooleanValue()) {
-							xml.append(1);
-						} else {
-							xml.append("<![CDATA[");
-							xml.append(r.getValue(d).getValueString(Locale.getDefault()));
-							xml.append("]]>");
+	private void saveIntoXML(WFCIndexPart indexPart, TreeList<Row> treeList, List<Integer> groupByIndices, List<ColumnInfo> colConfig,
+			List<Integer> columnReorderList, StringBuffer xml, boolean tabSeparated, String fileName, String title) {
+
+		// Viewport layer umgehen, damit in addSumRow() auf alle Zeilen zugegriffen werden kann
+		ILayer layer = indexPart.getBodyLayerStack().getUnderlyingLayerByPosition(2, 4).getUnderlyingLayerByPosition(2, 4);
+
+		NumberFormat numberFormat = NumberFormat.getInstance(Locale.getDefault());
+		numberFormat.setMinimumFractionDigits(2);
+		numberFormat.setMaximumFractionDigits(2); // TODO anpassen?
+
+		addHeader(xml, fileName);
+		xml.append("<Title>" + title + "</Title>\n");
+		xml.append("<IndexView>\n");
+		xml.append("<Group>\n" + "<Text><![CDATA[Gesamt]]></Text>\n"); // TODO: Übersetzen!
+
+		if (groupByIndices.isEmpty()) { // Keine Gruppierung
+			addRows(xml, treeList, colConfig, columnReorderList, numberFormat);
```

Comment:
Hier sollten wir noch den Translationservice bemühen. 

Propertie aus i18n heißt Total
`
<property name="Total">
                <default content="Total"/>
                <value content="Gesamt" lang="de"/>
                <value content="Total" lang="es"/>
                <value content="Total" lang="fr"/>
                <value content="Totale" country="CH" lang="it"/>
                <value content="Totale" lang="it"/>
            </property>
`